src/schema/updates/manifest.json

```diff
@@ -106,7 +106,7 @@
             "format": "versionString"
           },
           "manifest_version": {
-            "maximum": 2
+            "maximum": 3
```

Comment:
Could you fully remove the `manifest_version` object instead?

Reply:
Done